### DISCUSSION TITLE
How could I add a text to the social media Icon?

### DISCUSSION THREAD

**Discussion by aakashba** [alshedivat/al-folio#1991] [Q&A]
I would like to display some text above or below a social media Icon, like "My LinkedIn" above the LinkedIn social media icon. I do wanna move around the icon locations itself in the page so would like to tie to the icon itself.

**Comment by george-gca** [comment]
The specific location where we set the icons is in [_includes/social.html](https://github.com/alshedivat/al-folio/blob/master/_includes/social.html), which is used by [_layouts/about.html](https://github.com/alshedivat/al-folio/blob/master/_layouts/about.html). No idea which is the best way to achieve what you want, but I can at least point the direction.
